### PR TITLE
Feat: Roulette PoC UI

### PR DESCRIPTION
**Pull request by begonaalvarezd**
# Description of change

Implement the new UI for the Roulette PoC.

### CODE REVIEW

File: contracts/rust/fairroulette/frontend/src/components/betting_system.svelte [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  import Button from "./button.svelte";
+
+  let betAmount: number = 1;
+
+  $: betAmount, onBarChange();
+
+  $: isFirstStep = $bettingStep === BettingStep.NumberChoice;
+  $: isLastStep = $bettingStep === BettingStep.AmountChoice;
+
+  function onNumberClick(number): void {
```
Comment (lmoe) [inline contracts/rust/fairroulette/frontend/src/components/betting_system.svelte:19]: There is a strict type definition here which I like. Other functions don't have them. I think we should either remove them all or add them to all functions. Just to have a clean aligned code. 

/edit: also number should have a strict typing => `:number`

File: contracts/rust/fairroulette/frontend/public/index.html [hunk 2]
```diff
@@ -4,17 +4,31 @@
...
-	<link rel='icon' type='image/png' href='/favicon.png'>
+	<link rel='icon' type='image/png' href='/favicon.ico'>
 	<link rel='stylesheet' href='/global.css'>
 	<link rel='stylesheet' href='/build/bundle.css'>
-
+	<link href='https://webassets.iota.org/fonts/css/inter.css' rel='stylesheet' type='text/css' media='all'>
+	<link href='https://webassets.iota.org/fonts/css/metropolis.css' rel='stylesheet' type='text/css' media='all'>
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@500;700&display=swap" rel="stylesheet">
```
Comment (lmoe) [inline contracts/rust/fairroulette/frontend/public/index.html:27]: Do we have this `DM Sans` font as a webasset at iota.org as well? Then we can ditch the Google Font alltogether.

File: contracts/rust/fairroulette/frontend/src/components/selectors/bar.svelte [hunk 3]
```diff
@@ -0,0 +1,142 @@
+<script lang="ts">
+  import { updateFunds } from "../../lib/app";
+
+  import { balance } from "../../lib/store";
+
+  export let value: number;
```
Comment (lmoe) [inline contracts/rust/fairroulette/frontend/src/components/selectors/bar.svelte:6]: Same with the function typings. Let's try to keep it aligned. 
Either by defining a type everywhere, or remove it alltogether. (Personally, I prefer typings) :)

File: contracts/rust/fairroulette/frontend/src/lib/models/ILog.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+export type ILog = {
+    /**
+     * The label above the title.
```
Comment (lmoe) [inline contracts/rust/fairroulette/frontend/src/lib/models/ILog.ts:3]: Same with the typings, I would either have the documentation in all interfaces or remove them. In these cases the interfaces have proper properties. Personally I would get rid of them.

**Review by lmoe — APPROVED** [review]
Great to see what you made out of the existing project, really love it. 

Just added some minor German strictness remarks about code style. Otherwise LGTM! :)

**Comment by lmoe** [comment]
I will merge this for now. The mentioned comments will be incoorperated in an upcoming PR. 
Please still comment on this PR if you find things to correct. We will make sure to add them!

File: contracts/rust/fairroulette/frontend/src/components/betting_system.svelte (hunk 1 above)
Comment (begonaalvarezd) [reply contracts/rust/fairroulette/frontend/src/components/betting_system.svelte:19]: fixed in `boxfish-studio:feat/roulette-ui-updates`

File: contracts/rust/fairroulette/frontend/public/index.html (hunk 2 above)
Comment (begonaalvarezd) [reply contracts/rust/fairroulette/frontend/public/index.html:27]: Unfortunately we dont have DM Sans there

File: contracts/rust/fairroulette/frontend/src/components/selectors/bar.svelte (hunk 3 above)
Comment (begonaalvarezd) [reply contracts/rust/fairroulette/frontend/src/components/selectors/bar.svelte:6]: fixed in `boxfish-studio:feat/roulette-ui-updates`

**Pull request merged**